model-optimizer/mo/utils/utils.py

```diff
@@ -133,3 +135,16 @@ def unique_by(xs: list, predicate: Callable) -> list:
     """
     groups = group_by_with_binary_predicate(xs, predicate)
     return [group[0] for group in groups]
+
+
+def validate_json_config(json_config: list, config_file_name: str):
+    try:
+        base_dir = get_mo_root_dir()
+        schema_file = os.path.join(base_dir, 'mo', 'utils', 'schema.json')
+        with open(schema_file, 'r') as f:
+            schema = json.load(f)
+            validator = json_validate.compile(schema)
+            validator(json_config)
+    except Exception as exc:
+        raise Error("Failed to validate custom replacements configuration file '{}': {}. ".format(config_file_name, exc) +
+                    refer_to_faq_msg(70)) from exc
```

Comment:
please return boolean value from this function.
does validator or `compile` method have returned values? Please handle returned values if exist.

Reply:
Also, I think there is no need to move this function into `utils.py`. Please create this function in `custom_replacement_config.py`